lib/tree-view-package.js

```diff
@@ -7,6 +7,16 @@ const TreeView = require('./tree-view')
 module.exports =
 class TreeViewPackage {
   activate () {
+    const oldSort = atom.config.get('tree-view.sortFoldersBeforeFiles')
+    if (oldSort !== undefined) {
+      console.log('Migrating settings....')
+      atom.config.unset('tree-view.sortFoldersBeforeFiles')
+      if (oldSort) {
+        atom.config.set('tree-view.sortFolders', 'Before Files')
+      } else {
+        atom.config.set('tree-view.sortFolders', 'Alongside Files')
+      }
+    }
```

Comment:
This migration code should be extracted as a separate function (or method), and invoked here.